### PR TITLE
MTP-2169 refac dropdown component, new props, update stories

### PR DESCRIPTION
**Pull request by petterkoding**

### CODE REVIEW

**Review by lha047 — CHANGES_REQUESTED** [review]
Good job. Left some comments for improvements

File: mt-kit/core/css/src/global/_utils.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@
   }
 }
 
+.xxs-m {
```
[the remark this answers is not in the document]
Comment (petterkoding) [reply mt-kit/core/css/src/global/_utils.scss]: fixed

File: mt-kit/core/css/src/components/_dropdown.scss [hunk 2]
```diff
@@ -172,8 +174,48 @@
     }
   }
 
+  //currently used in dropdown filter on searchview
+  &.default-dropdown {
+    >.mt-button {
+      background-color: var(--color-mt-white);
```
[the remark this answers is not in the document]
Comment (petterkoding) [reply mt-kit/core/css/src/components/_dropdown.scss:177]: fixed

File: mt-kit/core/css/src/components/_dropdown.scss [hunk 3]
```diff
@@ -172,8 +174,48 @@
...
+      padding: var(--spacer-xx-small) var(--spacer-x-small);
+      width: 100%;
```
[the remark this answers is not in the document]
Comment (petterkoding) [reply mt-kit/core/css/src/components/_dropdown.scss:179]: i tried flex-basis, but it didnt work. However I removed the other width: 100%

File: mt-kit/core/css/src/components/_dropdown.scss [hunk 4]
```diff
@@ -172,8 +171,26 @@
     }
   }
 
+  //currently used in dropdown filter on searchview
```
Comment (lha047) [inline mt-kit/core/css/src/components/_dropdown.scss:174]: Remove comment

File: mt-kit/core/svelte/src/stories/Dropdown.stories.svelte [hunk 5]
```diff
@@ -79,7 +113,7 @@
         </ol>
       </Dropdown>
     </section>
-    <section class="layout-flex-col layout-flex-col--x-small" aria-labelledby="how-to-heading">
+    <section class="preview-wrapper layout-flex-col--x-small" aria-labelledby="how-to-heading">
```
Comment (lha047) [inline mt-kit/core/svelte/src/stories/Dropdown.stories.svelte:116]: Clean up `layout-flex-col--x-small`

**Review by lha047 — APPROVED** [review]
Good job :D See comments before merge

**Pull request merged**